assets/src/sass/utilities/_theme-supports.scss

```diff
@@ -4,8 +4,8 @@
 	filter: brightness(0);
 }
 
-// Support accessible color combinations for background colors.
-$color-palette-reverse:
+// Support accessible color combinations for dark background colors.
+$color-palette-reverse-dark:
```

Comment:
I was using "reverse" to mean white text on a dark background, so if we want to switch it to "light" and "dark", it would just be `$color-palette-dark` and `color-palette-light`.